src/main/java/com/rarchives/ripme/ripper/rippers/ChanRipper.java

```diff
@@ -19,7 +19,7 @@
 
 public class ChanRipper extends AbstractHTMLRipper {
     public static List<ChanSite> explicit_domains = Arrays.asList(
-        new ChanSite(Arrays.asList("boards.4chan.org"),   Arrays.asList("4cdn.org", "is.4chan.org")),
+        new ChanSite(Arrays.asList("boards.4chan.org"),   Arrays.asList("4cdn.org", "is.4chan.org", "is2.4chan.org")),
```

Comment:
My RegEx is sorta limited, but I know this'll do for the `isX.4chan.org`, accepting any number, and "defaults" to not having a number at all:

```
is(|[0-9]+)\.4chan\.org
```